src/pages/PageSearchResult.vue

```diff
@@ -21,10 +21,11 @@
         <h2>This page isn't available</h2>
         <p>An error occured while trying to load the search results</p>
         <p>
-          <small
-            >Please submit all bugs or issues to the project's GitHub issue
-            repository (link in footer).</small
-          >
+          <small>
+            Please submit all bugs or issues to
+            <a href="https://github.com/mozilla-iam/dino-park-issues">the
```

Comment:
Can we use the value from `LinksMixin` instead? You can see how to include a link in this commit: https://github.com/mozilla-iam/dino-park-front-end/commit/9dacc91357ae3b337337ac421742a99488d72cf1#

Reply:
Suggestion applied in both instances.

Thanks for the review!